apps/site-app/src/app.tsx

```diff
@@ -26,9 +26,8 @@ import { App } from 'containers/App';
 // Import Language Provider
 import { LanguageProvider, Languages } from '@thecointech/shared/containers/LanguageProvider';
 
-// Load the favicon and the .htaccess file
+// Load the favicon
 import '!file-loader?name=[name].[ext]!./images/favicon.ico';
-import 'file-loader?name=.htaccess!./.htaccess';
```

Comment:
there is no .htaccess on Firebase.  No point keeping it in our code base in that case.